pkg/driftctl.go

```diff
@@ -53,6 +54,15 @@ func (d DriftCTL) Run() (*analyser.Analysis, error) {
 		middlewares.NewAwsSNSTopicPolicyExpander(),
 	)
 
+	if !d.strictMode {
+		middleware = append(middleware,
+			middlewares.NewAwsIamPolicyAttachmentDefaults(),
```

Comment:
As we discussed in the issue, maybe a single middleware per cloud provider could be enough ? WDYT @wbeuil

Reply:
Yes I agree, once we redefine what we should put inside it, let's gather them inside one middleware.

Reply:
The method is not used anymore, so I reverted this change